CMakeLists.txt

```diff
@@ -39,38 +39,42 @@ elseif("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
   endif ()
 endif ()
 
-# Configure a header file to pass some of the CMake settings
-# to the source code
-configure_file(
-  "${PROJECT_SOURCE_DIR}/zeth_config.h.in"
-  "${PROJECT_BINARY_DIR}/zeth_config.h"
+# Default curve to use with zeth. Only useful for applications since all
+# library functions should be parameterized by type. Note, this is
+# intentionally independent of the libff variable `CURVE` to help ensure that
+# library code remains fully parameterized (and to avoid unnecessary rebuilds
+# of libff caused by changing compiler command line).
+set(
+  ZETH_CURVE
+  "ALT_BN128"
+  CACHE
+  STRING
+  "Default curve: one of BLS12_377, BW6_761, ALT_BN128, BN128, EDWARDS, MNT4, MNT6"
 )
 
-# Flags and compilation options to chose the type of zksnark
+# Configure a header file to pass some of the CMake settings
+# Option selecting the type of zksnark
```

Comment:
`Option selecting the type of zksnark` -> `Option selecting the zk-SNARK scheme`

Reply:
changed

Reply:
Thanks!